website_animate/static/src/js/o_animate.editor.js

```diff
@@ -5,12 +5,19 @@ var sOptions = require('web_editor.snippets.options');
 
 function forceAnimation() {
     var $els = $();
+    var $scrollingElement = $().getScrollingElement();
     _.each(arguments, function (el) {
         $els = $els.add(el);
     });
     $els.css('animation-name', 'dummy');
     void $els[0].offsetWidth;
+    $els.addClass('o_animating');
+    $scrollingElement.css({overflowX: 'hidden'});
```

Comment:
- Use a class for this or properly restore the original inline style overflow that a custo might have added.
- Stop using jQuery when you can easily